src/NServiceBus.AzureStoragePersistence.AcceptanceTests/app.config

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
```

Comment:
are the redirects really needed? just checking since often nuget incorrectly adds these for me